src/quo2/components/numbered_keyboard/keyboard_key/view.cljs

```diff
@@ -0,0 +1,34 @@
+(ns quo2.components.numbered-keyboard.keyboard-key.view
+  (:require [react-native.core :as rn]
+            [quo2.theme :as quo.theme]
+            [status-im.ui.components.icons.icons :as icons]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.numbered-keyboard.keyboard-key.style :as style]))
+
+(defn- get-label-color
+  [disabled theme blur]
+  (if disabled
+    (if (or (= :dark theme) blur)
+      colors/white-opa-30
+      colors/neutral-30)
+    (if (or (= :dark theme) blur)
+      colors/white
+      colors/neutral-100)))
+
+(defn- keyboard-key-internal
+  [{:keys [label? disabled theme blur? icon? on-press?]}]
+  (let [label-color (get-label-color disabled theme blur?)]
+    [rn/touchable-highlight
```

Comment:
hmm, I think we want to do this the same as the quo2 button component. That is we use the `rn/touchable-without-feedback` and have an internal state for `pressed` that we can set the background color on.

Reply:
Done